sass/px-dropdown.scss

```diff
@@ -58,11 +58,17 @@ $inuit-enable-btn--primary: true;
   border-color: var(--px-validation-success-text-color, green);
   color: inherit;
 }
-
 .dropdown-icon {
   --iron-icon-width: 1.06667rem;
   --iron-icon-height: 1.06667rem;
   margin-left: $inuit-base-spacing-unit--tiny;
+  /*
+    These margins are for pages that are rendered from "Right-To-Left" or "Left-To-Right"
+    If the writing direction is left-to-right, this value overrides margin-left.
+    If the writing direction is right-to-left, this value overrides margin-right.
+  */
+  -webkit-margin-start: $inuit-base-spacing-unit--tiny;
+  -moz-margin-start: $inuit-base-spacing-unit--tiny;
```

Comment:
What about IE and Edge?

Reply:
no support for these properties in IE/Edge - the content doesn't look bad, the only issue is that the carat for the dropdown is not spaced away from the text; it's still usable.

`normal dropdown in IE/Edge`
<img width="127" alt="dropdown-ie-edge" src="https://user-images.githubusercontent.com/30244294/52586458-6ade5c00-2dec-11e9-8f10-f0b06bb33642.png">

`RTL dropdown in ie/edge`
<img width="108" alt="rtl-dropdown-ie-edge" src="https://user-images.githubusercontent.com/30244294/52586457-6ade5c00-2dec-11e9-846a-cecff12b8ee8.png">